src/graph_manipulation/fwg_filter.py

```diff
@@ -0,0 +1,53 @@
+import networkx as nx
```

Comment:
Arra fel vannak készítve a metódusok, hogy valamelyik argumentum (nem az `fwg`) a `None` értéket kapja? Teszteld, hogy az is működik-e mindkét esetben.